script/cibuild

```diff
@@ -1,170 +1,76 @@
 #!/bin/bash
+#
```

Comment:
I personally would love to `set -ex` at the beginning of the file so we see what gets executed and we fail early. This way we see errors like this https://travis-ci.org/libgit2/objective-git/jobs/120803256 earlier and can debug it locally.